src/Alerts/Alert.ts

```diff
@@ -0,0 +1,46 @@
+export interface Alert {
+	id: number;
+	externalId: string;
+	perex: string;
+	type: Type;
+	severity: Severity;
+	affectedArea: GeoJSON.GeoJsonObject;
+	state: State;
+	summary: string;
+	validFrom: string;
+	validTo: string;
+	origin: string;
+	provider: string;
+	providerUrl: string;
+}
+
+export enum Type {
+	WIND = 'wind',
+	TORNADO = 'tornado',
+	RAIN = 'rain',
+	FOG = 'fog',
+	SNOW = 'snow',
+	BLIZZARD = 'blizzard',
+	FLOOD = 'flood',
+	AVALANCHE = 'avalanche',
+	STORM = 'storm',
+	SEVERE_WEATHER = 'severe weather',
+	EARTHQUAKE = 'earthquake',
+	ICING = 'icing',
+	COASTAL_HAZARDS = 'coastal hazards',
+	UNKNOWN = 'unknown'
+}
+
+export enum Severity {
+	MINOR = 'minor',
+	MODERATE = 'moderate',
+	SEVERE = 'severe',
+	EXTREME = 'extreme'
+}
+
+enum State {
+	ACTIVE = 'active',
+	INSUFFICIENT_DATA = 'insufficient data',
```

Comment:
Maybe we should remove insufficient_data alerts from api so we give out only ACTIVE, CANCELLED and OUTDATED and we can add filter for that. CANCELLED and OUTDATED are useful in case someone caches them. But this can be done later just thought...